config/localization/de/app.json

```diff
@@ -244,7 +244,7 @@
       "SLOW_WITHDRAW": "Langsame Auszahlung",
       "SORT_BY": "Sortieren nach",
       "SOURCE_ADDRESS": "Quelladresse",
-      "STAKED": "Stacked",
+      "STAKED": "Staked",
```

Comment:
reported from DoS by a native German speaker

Reply:
@mike-dydx  Can you ping Brenna on the @dydx_lilt Slack channel about this?  Otherwise, their system is gonna revert it back at some point in the future.